Mathlib/LinearAlgebra/TensorProduct.lean

```diff
@@ -94,12 +94,16 @@ def addMonoid : AddMonoid (M ⊗[R] N) :=
 protected instance add : Add (M ⊗[R] N) :=
   (addConGen (TensorProduct.Eqv R M N)).hasAdd
 
+instance zero : Zero (M ⊗[R] N) :=
+  (addConGen (TensorProduct.Eqv R M N)).addMonoid.toZero
+
 instance addZeroClass : AddZeroClass (M ⊗[R] N) :=
   { (addConGen (TensorProduct.Eqv R M N)).addMonoid with
```

Comment:
Why is this not `.addZeroClass` to start with?

Reply:
There's no `addZeroClass` instance on `AddCon`.

Reply:
Sorry, there is, but the file is in a funny order so I didn't see it.